app/styleguide/page.tsx

```diff
@@ -1037,23 +1059,161 @@ function Page() {
+</LayoutContainer>
+`}
+                </code></pre></div>
+              </ContentContainer>
+            </LayoutContainer>
+
+            <h3>CSS Variables</h3>
+            <dl>
+              <dt><code>--layout-container-padding</code></dt>
+              <dd>
+                Set the container padding, defaults to: <code>0</code>
+              </dd>
+
+              <dt><code>--layout-container-background</code></dt>
+              <dd>
+                Set the container background, defaults to: <code>transparent</code>
+              </dd>
+            </dl>
+
+            <h2><code>Content Container</code> (Base)</h2>
+            <ContentContainer style={{'border': '1px solid var(--slate-600)'}}>
+              <p>This conteint is wrapped in the following container.</p>
+              <p><strong>Note</strong> that the style property is not required
+                to use the container. It's only here to show that you can use it
+                this way, and to show the outline here in the styleguide.</p>
+              <div><pre><code>{`<ContentContainer style={{'border': '1px solid var(--purple-600)'}}>
+...
+</ContentContainer>`}
+              </code></pre></div>
+            </ContentContainer>
+
+            <h3>CSS Variables</h3>
+            <dl>
+              <dt><code>--layout-content-padding</code></dt>
+              <dd>
+                Set the content container padding, defaults to: <code>var(--brand-space2)</code>
+              </dd>
+
+              <dt><code>--layout-container-background</code></dt>
+              <dd>
+                Set the container background, defaults to: <code>transparent</code>
+              </dd>
+            </dl>
+
+            <h2><code>ToolbarContainer</code> (LayoutContainer)</h2>
+            <p><code>LayoutContainer > ToolbarContainer</code></p>
+            <ToolbarContainer style={{'--layout-container-background': 'var(--slate-200)'}}>
+              <p>Some toolbar text, maybe a Title</p>
+              <Button>Button</Button>
+            </ToolbarContainer>
+
+            <h3>CSS Variables</h3>
+            <p>Since this inherrits from <code>LayoutContainer</code> those CSS
+              variables also works. In addition to those, the following is also available:</p>
+            <dl>
+              <dt><code>--layout-gap</code></dt>
+              <dd>
+                This container uses flexbox, and this variable sets
+                the <code>gap</code> css property. Defaults to <code>1em</code>
+              </dd>
+            </dl>
+
+            <h2>Sidebar Container (Composite)</h2>
+            <p>This is a composite layout component than contains a sidebar
+              alongside a content container.</p>
+            <p><code>LayoutContainer > LayoutWithSidebar</code></p>
+            <p><code>ContentContainer > SidebarContainer</code></p>
+            <ToolbarContainer>
+              <Button onPress={toggleSidebarDirection}>Toggle Direction</Button>
```

Comment:
I noticed that on smaller mobile devices, the sidebar appears at the bottom of the page, and the toggling doesn't appear to be working.

Reply:
It's possible there was a misunderstanding somewhere, but I thought the component I was building, was for those pages with related content on the sidebar, and when we discussed that, people mentioned that on mobile the sidebar collapses below the content.

So this is what I've done here. I added the additional condition that if the direction is left, and it collapses, then it goes to the top instead of bottom.

Is this not what was intended? Because after our last discussion, I came to the conclusion I did the wrong thing originally, which is why I changed it.

Reply:
Ok. I thought you were building the "slider" component. We had talked about just using "Display:flex" for the right sidebar so that it will wrap below the content. In fact, I implemented the right sidebar in the Profile page that way.

But the "slider" component will slide in additional content, rather than always displaying it.